### ISSUE TITLE
File upload working in iOS 9.2.1, fails in iOS 9.3.1

### ISSUE THREAD

**Issue by StatLaw** [owncloud/ios#639]
### Steps to reproduce
1. Upgrade from iOS 9.2.1 to iOS 9.3.1
2. Try to upload a file from iphone to owncloud 9.0.1 using the app 3.4.8

### Expected behaviour
The file should upload.

Using iOS 9.2.1 one gets webserver log entries like:
`192.168.1.254 - - [02/May/2016:21:34:51 +0100] "GET /ocs/v1.php/apps/files_sharing/api/v1/shares?path=05%20-%20May/01/&subfiles=true HTTP/1.1" 200 741
192.168.1.254 - - [02/May/2016:21:34:51 +0100] "PROPFIND /remote.php/webdav/05%20-%20May/01/ HTTP/1.1" 207 1614
192.168.1.254 - - [02/May/2016:21:34:51 +0100] "PROPFIND /remote.php/webdav/05%20-%20May/01/Photo-2016-05-01-12-14-44_8697.JPG HTTP/1.1" 404 842
192.168.1.254 - - [02/May/2016:21:34:52 +0100] "PUT /remote.php/webdav/05%20-%20May/01/Photo-2016-05-01-12-14-44_8697.JPG HTTP/1.1" 201 667
192.168.1.254 - - [02/May/2016:21:35:00 +0100] "GET /ocs/v1.php/apps/files_sharing/api/v1/shares?path=05%20-%20May/01/&subfiles=true HTTP/1.1" 200 692
192.168.1.254 - - [02/May/2016:21:35:00 +0100] "PROPFIND /remote.php/webdav/05%20-%20May/01/ HTTP/1.1" 207 1570
192.168.1.254 - - [02/May/2016:21:35:00 +0100] "PROPFIND /remote.php/webdav/05%20-%20May/01/ HTTP/1.1" 207 2395
192.168.1.254 - - [02/May/2016:21:35:01 +0100] "GET /ocs/v2.php/apps/notifications/api/v1/notifications?format=json HTTP/1.1" 200 678
192.168.1.254 - - [02/May/2016:21:35:01 +0100] "GET /ocs/v1.php/apps/files_sharing/api/v1/shares?path=05%20-%20May/01/&subfiles=true HTTP/1.1" 200 665`

### Actual behaviour
The Uploads list in the app permanently shows "Waiting for server connection".

Using iOS 9.3.1 one gets webserver log entries limited to:
`192.168.1.254 - - [02/May/2016:21:34:51 +0100] "PROPFIND /remote.php/webdav/05%20-%20May/01/Photo-2016-05-01-12-14-44_8697.JPG HTTP/1.1" 404 842`

The app never sends the `PUT` command.

### Server configuration
**Web server:**
hiawatha 10.2
**Database:**
mysql
**PHP version:**
7.0.6
### Client
**iOS version:**
3.4.8
**ownCloud app version:**
9.0.1
**Device model:**
5
#### Web server error log
```
`192.168.1.254 - - [02/May/2016:21:34:51 +0100] "PROPFIND /remote.php/webdav/05%20-%20May/01/Photo-2016-05-01-12-14-44_8697.JPG HTTP/1.1" 404 842`
```
#### ownCloud log (data/owncloud.log)
```
Shows error for the PROPFIND command, but there is no evidence of any subsequent PUT.
```

**Comment by rperezb** [comment]
ey, can you please check this with the latest version, 3.4.9

On the other hand, the propfind error, it's not an error, it's the expected behavior; before uploading the file, it's checked whether a file with the same name exist on the path, if it doesn't exist -> 404 the upload process should go on

**Comment by StatLaw** [comment]
Agreed. The propfind error is not an error in the sense of being wrong, and it happens quite properly even when uploads work. That's what I intended to show in my example.

The latest version 3.4.9 of the owncloud app, released on 5 May, does indeed work with my setup. I just tested it with everything else unchanged (as specified above) and was able to upload my first set of 30 files in one go. Brilliant, thanks!
